src/stavenote.ts

```diff
@@ -1267,15 +1264,12 @@ export class StaveNote extends StemmableNote {
 
     // Apply the overall style -- may be contradicted by local settings:
     this.applyStyle();
-    this.setAttribute('el', ctx.openGroup('stavenote', this.getAttribute('id')));
+    ctx.openGroup('stavenote', this.getAttribute('id'));
     this.drawLedgerLines();
-    ctx.openGroup('note', undefined, { pointerBBox: true });
     if (shouldRenderStem) this.drawStem();
     this.drawNoteHeads();
     this.drawFlag();
     ctx.closeGroup();
-    this.drawModifiers();
-    ctx.closeGroup();
```

Comment:
Did you mean to remove drawModifiers here?

Reply:
Yes. I refactored `drawModifiers()` passing a `NoteHead` in order to be able to group NoteHead and Modifiers together. `drawModifiers(...)` is now called from `drawNoteHeads()` within the vf-notehead group.

This is what I meant with the second bullet above:
> - check that the groups are grouped consistently. ie.: modifier within notehead group

Reply:
@AaronDavidNewman Thanks a lot for the review! I am looking forward to getting the PRs through.

Reply:
@AaronDavidNewman are you fine with the explanation above? If this is ok and you approve the other changes, I will proceed and merge the PR